src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -687,8 +689,8 @@ export default class AircraftCommander {
         }
 
         if (spotInQueue > 0) {
-            readback.log = `number ${spotInQueue} behind ${aircraftAhead.callsign}`;
-            readback.say = `number ${spotInQueue} behind ${aircraftAhead.getRadioCallsign()}`;
+            readback.log = `number ${spotInQueue} for departure`;
+            readback.say = `number ${spotInQueue} for departure`;
```

Comment:
To preserve this functionality (since it is quite useful), @n8rzz would you be opposed to the slight ickiness of the following resolution below?

![image](https://user-images.githubusercontent.com/5103735/141342829-33c013f7-eca6-40c9-9b7f-72276385011d.png)

Reply:
done